crates/sui-network/build.rs

```diff
@@ -31,6 +31,15 @@ fn main() -> Result<()> {
                 .codec_path(codec_path)
                 .build(),
         )
+        .method(
+            Method::builder()
+                .name("transaction_v2")
+                .route_name("TransactionV2")
+                .input_type("sui_types::messages_grpc::HandleTransactionRequestV2")
+                .output_type("sui_types::messages_grpc::HandleCertificateResponseV3")
```

Comment:
Why wouldn't there be a signature with the effects?

Reply:
Because this handler is for fastpath, which does not produce BLS signatures per effects which is needed for `SignedTransactionEffects` included in `HandleCertificateResponseV3`.

Reply:
I might be missing some context, but why doesn't it produce a signature per effects?

Reply:
During the discussions we had previously, it was determined that effects certificate was unnecessary at the beginning. So not producing the effects BLS signatures to save CPU seems to be the right choice.

Reply:
How does the caller verify that a quorum of validators has approved the result without the effects sig?

Reply:
I guess they couldn't have done that with just one authority sig anyway but I'm curious how it works right now.

Reply:
Discussed with @lxfind offline. The critical thing is that we will need to support TLS for transaction handler.

Reply:
I would consider supporting TLS for tx handler pretty high priority regardless of anything to do with fast path

Reply:
Great! That would be a good follow up.